src/main/resources/application.conf

```diff
@@ -50,4 +50,8 @@ user-events {
   # Enables KamonRecorder so as to enable sending metrics to Kamon supported backends
   # like DataDog
   enable-kamon = false
+
+  # Namespaces that should not be monitored
+  blacklisted-namespaces = ["guest"]
```

Comment:
Instead of blacklisted `ignored-namespaces` may be better? As blacklisted indicates that such namespaces are disabled but intention here is more to ignore certain namespaces

Further the default value we should keep as empty and instead specify the value on deployment side